ttnn/cpp/ttnn/operations/pool/maxpool/maxpool_pybind.hpp

```diff
@@ -9,13 +9,14 @@
 
 #include "ttnn/cpp/pybind11/decorators.hpp"
```

Comment:
Please split to .hpp and .cpp
This is the best example we got so far
https://github.com/tenstorrent/tt-metal/blob/main/ttnn/cpp/ttnn/operations/ccl/all_gather/all_gather_pybind.hpp

This will help keep build time lower